internal-services/catalog/iib-add-fbc-fragment-to-index-image-task.yaml

```diff
@@ -157,19 +179,29 @@ spec:
              "fbc_opt_in": $fbc_opt_in,
              "publish_index_image": $publish_index_image,
              "sign_index_image": $sign_index_image
-            } | tostring' | tee $(results.genericResult.path)
+            } | tostring' | tee "$(results.genericResult.path)"
+
+        # if it finds a build which is completed or in progress, it should exit this step and jump to
+        # the next step `s-wait-for-build-state` which will watch the build until it is completed.
+        build=$(check_previous_build "${KRB5_PRINCIPAL}" "$(params.fromIndex)" "$(params.fbcFragment)")
+        if [ -n "${build}" ]; then
```

Comment:
You have `// empty` on the `jq` command in `check_previous_build`. Won't this always be true?
```
$ foo=$(jq '.foo // empty' <<< "{}")
$ if [ -n $foo ] ; then echo hi ; fi
hi
```

Reply:
cool. Just tested it and it is working fine. I am resolving it, thanks!